mathics/builtin/base.py

```diff
@@ -763,10 +775,8 @@ def is_literal(self) -> bool:
         depend on definition bindings. That is why, in contrast to
         `is_uncertain_final_definitions()` we don't need a `definitions`
         parameter.
-
-        Think about: We will say that a BoxExpression can't change.
         """
-        return True
+        return False
```

Comment:
Why a `BoxExpression` shouldn´t be handled as a literal?

Reply:
From line 187 in this PR of element.py: 

> - *literal_values - optional: if this is not None, then all elements are (Python) literal values and literal_value  contains these literals.

Reply:
OK, I see.